roles/openldap/defaults/main.yml

```diff
@@ -0,0 +1,52 @@
+keystone:
+  ldap:
+    enabled: False 
+
+    # ldap server settings
+    url: ldap://0.0.0.0:389
```

Comment:
Although this can be done the guide here http://www.openldap.org/devel/admin/security.html recommends using firewall configuration to restrict access.